Description: Replace two paypal images with text
 This avoids making external requests.
 .
Author: Christopher Baines <mail@cbaines.net>
Last-Update: 2015-10-25

--- scikit-learn-0.16.1.orig/doc/about.rst
+++ scikit-learn-0.16.1/doc/about.rst
@@ -110,14 +110,7 @@ for code sprints, as well as towards the
     target="_top">
     <input type="hidden" name="cmd" value="_s-xclick">
     <input type="hidden" name="hosted_button_id" value="74EYUMF3FTSW8">
-    <input type="image"
-    src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif"
-    border="0" name="submit" alt="PayPal - The safer, easier way to pay
-    online!" style="position: relative;
-    left: 40%;">
-    <img alt="" border="0"
-    src="https://www.paypalobjects.com/en_US/i/scr/pixel.gif" width="1"
-    height="1">
+    <input type="submit" value="PayPal">
     </form>
     </br>
 
